platform/administer/platform-database/database.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Platform Database
+sidebar_label: Database
+sidebar_position: 1
+description: Information that an admin may need to know about the platform database.
+---
+
+import BasePrerequisites from '../../_partials/install/base-prerequisites.mdx';
+
+import InstallNextSteps from '../../_partials/install/install-next-steps.mdx';
+
+import VersionBadge from '@site/src/components/VersionBadge';
+
+<VersionBadge platformVersion="v4.4.0"/>
+
+The platform database is a local postgres database server that is used and managed by vCluster Platform.
+
+## Configure
+
+By default, platform database will leverage the default StorageClass. If you would like to specify a different StorageClass, you can navigate to `Admin > Config` and edit the  `platformDB.storageClass` field.
+
+### Troubleshoot
+
+The PersistentVolumeClaim. that backs the platform database should never be deleted. It is named `platform-db`. In the case that it is deleted, the following steps can be followed to return the platform database to a working state.
+
+1. Set $PLATFORM_NS and $PLATFORM_NAME, based on Deployment or StatefulSet name for vCluster Platform.
```

Comment:
There's a `<InterpolatedCodeBlock />` component now, to allow readers to tweak those values and have them updated in the code block for copying.

Reply:
Cool, thanks for pointing that out!